tools/prometheus/prometheus.yml

```diff
@@ -9,15 +9,21 @@ alerting:
   alertmanagers:
     - scheme: http
       timeout: 10s
-      api_version: v1
+      api_version: v2
       static_configs:
         - targets: []
 scrape_configs:
-  - job_name: psp_api
+  - job_name: psp-api
     honor_timestamps: true
-    scrape_interval: 10s
     metrics_path: /metrics
     scheme: http
     static_configs:
       - targets:
           - host.docker.internal:5000
+  - job_name: otel-collector
+    honor_timestamps: true
+    metrics_path: /metrics
+    scheme: http
+    static_configs:
+      - targets:
+          - otel-collector:9464
```

Comment:
We need to instruct our local prometheus container to scrape the metrics generated by otel-collector (from the traces that are sent to it by our React frontend). 

The following image shows a diagram of our setup for local monitoring:

![otel-collector-jaeger-prometheus-architecture-2](https://github.com/user-attachments/assets/8bcecc83-c7f6-49c6-a97f-d7d57984c9c8)